subsys/bluetooth/mesh/rpl.h

```diff
@@ -0,0 +1,22 @@
+/*  Bluetooth Mesh */
+
+/*
+ * Copyright (c) 2017 Intel Corporation
+ * Copyright (c) 2020 Lingao Meng
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+struct bt_mesh_rpl {
+	uint16_t src;
+	bool  old_iv;
+	uint32_t seq;
+};
+
+void bt_mesh_rpl_reset(void);
+bool bt_mesh_replay_check(struct bt_mesh_net_rx *rx,
+			struct bt_mesh_rpl **match);
+void bt_mesh_rpl_clear(void);
+void bt_mesh_update_rpl(struct bt_mesh_rpl *rpl,
+			struct bt_mesh_net_rx *rx);
+struct bt_mesh_rpl *bt_mesh_rpl_alloc(void);
```

Comment:
Looks like you inherited a mix of terminology and function name formats here. Could you perhaps align all functions to be `bt_mesh_rpl_<...>`, for consistency?